src/main/java/placement/model/Fleet.java

```diff
@@ -0,0 +1,39 @@
+package placement.model;
+
+import placement.model.ship.Ship;
+import placement.model.ship.ShipType;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static placement.model.ship.ShipType.SUBMARINE;
+import static placement.model.ship.ShipType.DESTROYER;
+import static placement.model.ship.ShipType.CRUISER;
+import static placement.model.ship.ShipType.BATTLESHIP;
+
+/**
+ * Represents a default fleet of ships.
+ */
+public final class Fleet {
+    private List<Ship> shipList;
+
+
+    private Fleet(ShipType... shipTypes) {
+        shipList = new ArrayList<>();
+        for (ShipType shipType : shipTypes) {
+            Ship ship = new Ship(shipType.getSize());
+            shipList.add(ship);
+        }
+    }
+
+    public static Fleet createDefaultFleet() {
+        return new Fleet(BATTLESHIP,
+                CRUISER, CRUISER,
+                DESTROYER, DESTROYER, DESTROYER,
```

Comment:
A would suggest changing names of ships according amount of masts e.g.  FourMasts.